modelcards/cards.py

```diff
@@ -104,6 +105,38 @@ def load(cls, repo_id_or_path: Union[str, Path]):
 
         return cls(Path(card_path).read_text())
 
+    def validate(self, repo_type=None):
```

Comment:
Should we not set the default as it is?

```suggestion
    def validate(self, repo_type="model"):
```